README.md

```diff
@@ -8,6 +8,24 @@ This is not an official Google product.
 
 Author: David Tomaschik <dwt@google.com>
 
+
+### Installation using Docker###
```

Comment:
White space required after Docker before ### to render as end of H3 tag.  (Otherwise, rendered as 3 literal #)